packages/react-components/src/Table/Head.tsx

```diff
@@ -124,6 +124,11 @@ export default React.memo(styled(Head)`
     &.start {
       text-align: left;
     }
+
+    &.balances {
+      text-align: right;
+      padding-right: 218px;
```

Comment:
Avoid using `px` in favour of `rem`. Also, I wonder if this styling could possibly go closer to Accounts.

Reply:
@MiZiet

Reply:
Changed px to rem.